packages/drivers/http/src/http.js

```diff
@@ -1,5 +1,7 @@
 // TODO: once Object.fromEntries reach the JS Standard, remove this
 const fromEntries = iterable => Object.assign({}, ...Array.from(iterable, ([key, val]) => ({ [key]: val })))
+// extract filename of content-disposition
+const getFilename = string => /filename="(.*)"/g.exec(string)[1]
```

Comment:
Look for wikipedia + msdn if this is a standard or not

Reply:
this is a standard for file download or upload.
if we have multi file, `content-type` become `multipart/form-data`